### ISSUE TITLE
Gallery - `NavigationView` jumping on selection

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#16592]
### Current behavior

![scrollonselect](https://github.com/unoplatform/uno/assets/1075116/6ca010af-a633-4726-9a7f-66c647f041db)

### Works on UWP/WinUI

None

### Affected platforms

WebAssembly

**Comment by sakshi173** [comment]
@agneszitte this issue is already raised long back.

**Comment by MartinZikmund** [comment]
Can confirm it works in canary!

**Comment by agneszitte** [comment]
> Can confirm it works in canary !

@MartinZikmund are you still able to reproduce it in prod please
As if it is fixed with Canary is great but if it is still visible on prod, we should update prod

**Comment by MartinZikmund** [comment]
@agneszitte yes, still in prod. Not sure if this issue is necessary though, as it is not something to fix in code, but rather just deployment concern 🤔
